code/exercises/affine-cipher/affine-cipher.scm

```diff
@@ -0,0 +1,7 @@
+(import (rnrs))
+
+(define (encode key text)
+  'implement-me!)
+
+(define (decode key text)
+  'implement-me!)
```

Comment:
I did these by hand, because the `stubs` field didn't seem to do anything. Unless I was supposed to make it a pair with a list as the second part. IDK.

Reply:
Yeah unfortunately this is done by hand pretty much, for the time being. It could probably be improved by scanning the `property` field in the canonical file

Reply:
I don't think it is a big deal for now. I just thought I was doing something wrong 😀

Reply:
Yeah I think it can be improved in the future, for now it's not the end of the world haha